client/settings/express-checkout/express-checkout-button-preview.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * External dependencies
+ */
+import { React, useState, useEffect } from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+import {
+	PaymentRequestButtonElement,
+	useStripe,
+} from '@stripe/react-stripe-js';
+
+/**
+ * Internal dependencies
+ */
+import { shouldUseGooglePayBrand } from './utils/utils';
+import InlineNotice from '../../components/inline-notice';
+
+/* Will be used once hooked up to data persistence.
+import {
+	usePaymentRequestButtonType,
+	usePaymentRequestButtonSize,
+	usePaymentRequestButtonTheme,
+} from '../../data';
+*/
+
+/**
+ * stripePromise is used to pass into <Elements>'s stripe props.
+ * The stripe prop in <Elements> can't be change once passed in.
+ * Keeping this outside of <ExpressCheckoutButtonPreview> so that
+ * re-rendering does not change it.
+ */
+
+const BrowserHelpText = () => {
+	let browser = 'Google Chrome';
+	let paymentMethodName = 'Google Pay';
+
+	if ( shouldUseGooglePayBrand() ) {
+		browser = 'Safari';
+		paymentMethodName = 'Apple Pay';
+	}
+
+	return (
+		<p className="payment-method-settings__preview-help-text">
+			{ sprintf(
+				__(
+					/* translators: %1: Payment method name %2: Browser name. */
+					'To preview the %1$s button, view this page in the %2$s browser.',
+					'woocommerce-gateway-stripe'
+				),
+				paymentMethodName,
+				browser
+			) }
+		</p>
+	);
+};
+
+const buttonSizeToPxMap = {
+	default: 40,
+	medium: 48,
+	large: 56,
+};
+
+const ExpressCheckoutButtonPreview = () => {
```

Comment:
What do you think about keeping the name `PaymentRequestButtonPreview` for this component? The express checkout options are still called payment request buttons on the back-end (and will probably remain so). This will keep naming consistent with WCPay.